.github/ISSUE_TEMPLATE/new_component.md

```diff
@@ -0,0 +1,24 @@
+---
+name: New component proposal
+about: Suggest a new component for the project
+title: 'New component: '
+labels: new component
+assignees: ''
+
+---
+
+**The purpose and use-cases of the new component**
+
+*This information can be used later on to populate the README for the component. See an example overview [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/awsecscontainermetricsreceiver#overview).*
```

Comment:
```suggestion
<!--
*This information can be used later on to populate the README for the component. See an example overview [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/awsecscontainermetricsreceiver#overview).*
-->
```

Any reason to not make these commented blocks in the issue template? Might help ensure submitters keep the header (consider making a `##`) and remove this block to reduce noice in submission

Reply:
Good call, done!